ONSdigital/census-rm-case-api-2021
# CLERICAL_ADDRESS_RESOLUTION was missing in event types list

# Motivation and Context
Case API needs to have 100% of the event types that could be in the DB, otherwise it will have issues when it tries to load cases which have certain events.

# What has changed
Added missing CLERICAL_ADDRESS_RESOLUTION.

# How to test?
Create a case with CLERICAL_ADDRESS_RESOLUTION event. Check that you can see the event in ROps UI.

# Links
Trello: https://trello.com/c/eTwdornT
